extensions/src/main/scala/pl/touk/nussknacker/extensions/service/SqlEnricher.scala

```diff
@@ -0,0 +1,114 @@
+package pl.touk.nussknacker.extensions.service
+
+import pl.touk.nussknacker.engine.api._
+import pl.touk.nussknacker.engine.api.context.ProcessCompilationError.CustomNodeError
+import pl.touk.nussknacker.engine.api.context.transformation.{DefinedEagerParameter, NodeDependencyValue, SingleInputGenericNodeTransformation}
+import pl.touk.nussknacker.engine.api.context.{ProcessCompilationError, ValidationContext}
+import pl.touk.nussknacker.engine.api.definition._
+import pl.touk.nussknacker.engine.api.test.InvocationCollectors
+import pl.touk.nussknacker.engine.api.typed.typing.{Typed, TypingResult}
+import pl.touk.nussknacker.engine.api.typed.{TypedMap, typing}
+import pl.touk.nussknacker.extensions.db.WithDBConnectionPool
+import pl.touk.nussknacker.extensions.db.schema.TableDefinition
+
+import java.sql.ParameterMetaData
+import javax.sql.DataSource
+import scala.collection.mutable.ArrayBuffer
+import scala.concurrent.{ExecutionContext, Future}
+
+object SqlEnricher {
+
+  final val ArgPrefix: String = "arg"
+
+  final val QueryParamName: String = "Query"
```

Comment:
You can make Query Param as `SimpleEditorType.SQL_EDITOR`. Example you can see: https://staging.nussknacker.io/visualization/sql-editor?businessView=false&nodeId=sql-source

Reply:
:+1: I've made it yesterday :)
https://github.com/TouK/nussknacker/pull/1451/commits/ff25d3d8f99e904547640cf57b3d74f3d254809e